packages/zoe/src/instanceRecordStorage.js

```diff
@@ -39,7 +42,8 @@ export const makeInstanceRecordStorage = () => {
     );
 
     assertInstantiated();
-    instanceRecord = {
+    const instanceRecord = baggage.get('instanceRecord');
+    const nextInstanceRecord = {
```

Comment:
```suggestion
    const nextInstanceRecord = harden({
```

This should always have had a `harden` on it. Not sure why it didn't. All our stores will only store passables. Unfortunately, to ease the transition, they harden their arguments before checking if they're passable, thereby masking bugs like the missing harden here. Ideally, the stores would only check that things were passable, rejecting non-hardened things, and objects would be hardened as early a possible, usually when first created, as here.

Reply:
done